packages/loot-core/src/client/update-notification.js

```diff
@@ -0,0 +1,20 @@
+export default async function checkForUpdateNotification(
+  addNotification,
+  getIsOutdated
+) {
+  let isOutdated = await getIsOutdated();
+  if (!isOutdated) return;
+  addNotification({
+    type: 'message',
+    title: 'A new version of Actual is available!',
+    message: 'Click here to learn more',
```

Comment:
Since you can’t actually click this text to learn more, could you change this to “Version `${version}` of Actual was recently released.” (or something like that)